agent-core/src/test/java/com/newrelic/agent/android/analytics/AnalyticsEventTests.java

```diff
@@ -151,4 +151,10 @@ public void getMutableAttributeSet() {
             Assert.fail("Attributes should be mutable");
         }
     }
+
+    @Test
+    public void testEventUUID() {
+        AnalyticsEvent event = new AnalyticsEvent("testEvent");
+        Assert.assertNotNull(event.getEventUUID());
```

Comment:
If it's really a UUID, you should test that it parses into a new UUID. Take a look at `UUID.fromString()`.